app.js

```diff
@@ -14,78 +14,106 @@ const teamTwoLabel = document.getElementById('team-two-name');
 
 // create an array to hold on to the state of past games
 
-let name1 = '',
-let name2 =  '',
-let score1 = 0,
-let score2 = 0,
+let name1 = '';
+let name2 = '';
+let score1 = 0;
+let score2 = 0;
+
+const gamesArray = [];
 
 nameForm.addEventListener('submit', (e) => {
     // don't forget to prevent the default form behavior!
-
+    e.preventDefault();
     // get the name data from the form
-
+    const data = new FormData(nameForm);
     // set the state to this data from the form
-
+    name1 = data.get('team-name-1');
+    name2 = data.get('team-name-2');
```

Comment:
look at the `name` of your inputs in index.html.  This should match what you are passing into `data.get` on lines 30 and 31